notebooks/search/00-quick-start.ipynb

```diff
@@ -135,19 +135,51 @@
     "If you're running Elasticsearch locally or self-managed, you can pass in the Elasticsearch host instead. [Read more](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/connecting.html#_verifying_https_with_certificate_fingerprints_python_3_10_or_later) on how to connect to Elasticsearch locally."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "cb6ad7e9-0636-4cf3-a803-bf160fe16b33",
+   "metadata": {},
+   "source": [
+    "### Enable Telemetry\n",
+    "\n",
+    "Knowing that you are using this notebook helps us decide where to invest our efforts to improve our products. We would like to ask you that you run the following code to let us gather anonymous usage statistics. See [telemetry.py](https://github.com/elastic/elasticsearch-labs/blob/main/telemetry/telemetry.py) for details. Thank you!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b04f442-729d-406d-b826-654483498df6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!curl -O -s https://raw.githubusercontent.com/elastic/elasticsearch-labs/main/telemetry/telemetry.py\n",
+    "!curl -O -s https://raw.githubusercontent.com/miguelgrinberg/elasticsearch-labs/notebook-telemetry/telemetry/telemetry.py\n",
+    "from telemetry import enable_telemetry\n",
+    "\n",
+    "enable_telemetry(client)"
```

Comment:
I'm wondering if we should just explicitly add the names in the code. It makes it more obvious what exactly we track, which users might appreciate. And we would remove some of the uncertainty of the extraction failing with future versions.

Reply:
My only problem is that it requires more maintenance. Every notebook will have to be hand-edited. The goal of auto-determining the name is that you have a snippet that you copy/paste into a new notebook and you are done. Also a visible name may lead to some people editing the name, so we are going to get some amount of bogus names, probably.

Reply:
with the combination of the way we derive the name (which I think will need careful maintenance aswell) and every notebook needs to be edited anyway to include this script, i personally would rather this be explicit.

Reply:
I'm not convinced, but that's fine, we can start with explicit names and reconsider in the future if we get annoyed having to carefully update all notebooks without messing up the names.